client/packages/host/src/components/AppDrawer/SyncNavLink.tsx

```diff
@@ -18,8 +18,17 @@ export const SyncNavLink = () => {
     POLLING_INTERVAL_IN_MILLISECONDS
   );
 
+  // the Badge does not show if the content is 0
+  // somehow though the numberOfRecordsInPushQueue can be '0' which does show
+  const badgeContent = (
+    !numberOfRecordsInPushQueue
+      ? 0
+      : // parse, after casting to string to satisfy ts
+        Number.parseFloat(`${numberOfRecordsInPushQueue}`)
```

Comment:
You can just do `Number(numberOfRecordsInPushQueue)`, which also avoids having to do the explicit string conversion.

Is there any difference?

Reply:
thanks, yeah. not an important difference in this case and it has reminded me to check for NaN too. 
have changed it up.